.github/workflows/ci.yml

```diff
@@ -0,0 +1,81 @@
+name: build and consume by foreman & plugins
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - repo: foreman
+            org: theforeman
+            node-version: 10.x
+          - repo: katello
+            org: Katello
+            node-version: 10.x
+          - repo: foreman-tasks
+            org: laviro
+            ref: feat/foreman_js_link
+            node-version: 10.x
+          - repo: foreman
+            org: theforeman
+            node-version: 12.x
+          - repo: katello
+            org: Katello
+            node-version: 12.x
+          - repo: foreman-tasks
+            org: laviro
+            ref: feat/foreman_js_link
+            node-version: 12.x
```

Comment:
any idea how to reduce the duplication and set a matrix also for the node versions?

Reply:
Wouldn’t something like this work?
```yaml
matrix:
  node-version: [10, 12]
  repo: [theforeman/foreman, Katello/katello, theforeman/foreman-tasks]
```

Reply:
I tried that, but if I recall it generated only 2 jobs ( node 10, node 12)